### ISSUE TITLE
Expense Report Decimal Places

### ISSUE THREAD

**Issue by DH88UK** [Dolibarr/dolibarr#16993]
I just noticed that on the expense report (draft status) the numeric value in the U.P 'net' column is not shown with following zeroes when the value ends with zeroes.

this is different to the U.P 'inc tax' column and both 'amount' columns which shows the ending zeroes

<img width="128" alt="2021-03-29 15_33_22-Expense report - Card and 2 more pages - Personal - Microsoft​ Edge" src="https://user-images.githubusercontent.com/58177104/112852920-4b761c00-90a4-11eb-835f-dc83fa5d0610.png">

**Comment by ksar-ksar** [comment]
Hi,

Could you please fill the template : 


## Environment
- **Version**: [*Affected Dolibarr version(s)*]
- **OS**: [*Server OS type and version*]
- **Web server**: [*Webserver type and version*]
- **PHP**: [*PHP version*]
- **Database**: [*Database type and version*]
- **URL(s)**: [*Affected URL(s)*]

**Comment by DH88UK** [comment]
Version: 13.0.1
OS: Linux Mint 20.1
Web server: Apache/2.4.41 (Ubuntu)
PHP: 7.4.3
Database: MariaDB 10.3.25
URL(s): /expensereport/card.php?id=19

**Comment by ksar-ksar** [comment]
Same for me on 13.0.2
![image](https://user-images.githubusercontent.com/35605507/112857639-58e5d300-90b1-11eb-8647-05613fa74328.png)

**Comment by DH88UK** [comment]
Eventually i did some more research into this issue and found the problem in the block starting line 2093 of card.php

Corrected block:

`// Unit price HT`
`print '<td class="right">';`
`if (!empty($line->value_unit_ht))`
`{`
`print price($line->value_unit_ht);`
`} else {`
`$tmpvat = price2num(preg_replace('/\s*\(.*\)/', '', $line->vatrate));`
`$pricenettoshow = price2num($line->value_unit / (1 + $tmpvat / 100), 'MU');`
`print price($pricenettoshow);`
`}`

in summary, the price() function needs to be used
$pricenettoshow;
becomes
**price**($pricenettoshow);

**Comment by DH88UK** [comment]
Looks like this was already fixed in 15.0
